src/oauth2.js

```diff
@@ -25,7 +25,7 @@ const listenForCredentials = (popup, state, resolve, reject) => {
     if (response.access_token) {
       const result = {
         token: response.access_token,
-        expires_at: response.expires_in ? new Date().getTime() + parseInt(response.expires_in) * 1000 : null
+        expiresAt: response.expires_in && !isNaN(response.expires_in) ? new Date().getTime() + parseInt(response.expires_in) * 1000 : null
```

Comment:
Yeah, I just want to be defensive about what the auth server returns. However now I see another Problem:
```
isNaN("") === false
isNaN(parseInt("")) === true
```
We should probably do `parseInt` first, and then `isNaN`. Maybe something like (pseudo code off the top of my head):

```
const expiresIn = response.expires_in ? parseInt(response.expires_in) : NaN
const result = {
  token: response.access_token,
  expiresAt: isNaN(expiresIn) ? null : new Date().getTime() + expiresIn * 1000
}
```